src/domain/safe/entities/schemas/safe-list.schema.ts

```diff
@@ -0,0 +1,10 @@
+import { JSONSchemaType } from 'ajv';
+import { SafeList } from '../safe-list.entity';
+
+export const safeListSchema: JSONSchemaType<SafeList> = {
+  type: 'object',
+  properties: {
+    safes: { type: 'array', nullable: true, items: { type: 'string' } },
```

Comment:
Why is this nullable? I tried to request Safes for an non-owner address and it seems the transaction service returns an empty array, but not null...

Reply:
Done here https://github.com/5afe/safe-client-gateway-nest/pull/185/commits/9ec8ddf48b53db0303637cde5bd984f330eb67a6